flocker/node/_change.py

```diff
@@ -27,6 +29,11 @@ class IStateChange(Interface):
     """
     An operation that changes local state.
     """
+    eliot_action = Attribute(
```

Comment:
The doc isn't accurate; usage indicates this is supposed to return a *new* *started* action. So really this should be a method, not an attribute.